dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -36,7 +36,7 @@ enum bit<16> dash_tunnel_dscp_mode_t {
 
 struct eni_data_t {
     bit<32> cps;
-    bit<32> pps;
+    bit<32> bw;
```

Comment:
although bw is the one we use in the HLD, I wonder if we like to get the unit clear, such as bw_in_kbps.

Reply:
@r12f good idea. At least PPS is unambiguous. The item name should include the units to make it clear.

Reply:
All BW throughout SAI is in Mb/s

Reply:
why not keep it `bw` and bw can accept values like 978, 1K, 1M, 15G, 2T, (if we dont want toype too many zeroes) and the last char if letter detonates the unit this way we don't end up with 20 api names, like `bw_in_3_and_a_half_bytes_increments` (exaggeration to make the point)